packages/core/src/ProgressBar/ProgressBar.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Flex, Box } from 'pcln-design-system'
+
+const CustomBox = styled(Box)`
+  width: 100%;
+  border-radius: 2px;
+  margin-right: 4px;
```

Comment:
for width you can use it as a prop on the component `width={1}`
for margin-right you can use the prop `mr={1}` to achieve the same effect